nephos/fabric/utils.py

```diff
@@ -17,7 +17,7 @@
 
 from kubernetes.client.rest import ApiException
 
-from nephos.helpers.k8s import Executer, secret_create, secret_from_file, secret_read
+from nephos.helpers.k8s import Executer, secret_create, secret_read, secret_from_files
```

Comment:
Just a comment, I usually put these in alphabetic order, but you can correct in next PR.